test/http_client.cpp

```diff
@@ -6,7 +6,8 @@
 TEST(siesta, client_connect_ok)
 {
     auto f = siesta::client::getRequest("http://postman-echo.com/get?foo1=bar1",
-                                        5000);
+        std::vector<std::tuple<std::string, std::string>>(),
```

Comment:
Since a header item is a key+value pair, I'd recommend using std::pair instead of std::tuple for clarity.

Reply:
Ok i updated (see newer commit)

improved exemples & src code